stackit/internal/services/iaas/iaas_acc_test.go

```diff
@@ -44,13 +49,26 @@ var networkAreaRouteResource = map[string]string{
 var volumeResource = map[string]string{
 	"project_id":        testutil.ProjectId,
 	"availability_zone": "eu01-1",
-	"name":              "name",
+	"name":              fmt.Sprintf("tf-acc-%s", acctest.RandStringFromCharSet(5, acctest.CharSetAlpha)),
 	"description":       "description",
 	"size":              "1",
 	"label1":            "value",
 	"performance_class": "storage_premium_perf1",
 }
 
+// Server resource data
+var serverResource = map[string]string{
+	"project_id":        testutil.ProjectId,
+	"availability_zone": "eu01-1",
+	"size":              "64",
+	"source_type":       "image",
+	"source_id":         testutil.IaaSImageId,
+	"name":              fmt.Sprintf("tf-acc-%s", acctest.RandStringFromCharSet(5, acctest.CharSetAlpha)),
+	"machine_type":      serverMachineType,
+	"label1":            "value",
+	"user_data":         "#!/bin/bash",
```

Comment:
In the description it says it must be base64 encoded, isn't it a requirement on the API side then?

Reply:
Ups, I need to update the description. We decided to use a string value on the field and we do the conversion in the provider logic